slack_pull_reminder.py

```diff
@@ -26,14 +33,19 @@
 
 
 def fetch_repository_pulls(repository):
-    return [pull for pull in repository.pull_requests()
-            if pull.state == 'open']
+    pulls = []
+    for pull in repository.pull_requests():
+        if pull.state == 'open':
+            if USERNAMES and pull.user.login.lower() in USERNAMES:
```

Comment:
what happens if usernames is empty here? shouldn't it be:

```python
if pull.state == 'open' and (not USERNAMES or pull.user.login.lower() in USERNAMES):
  pulls.append(pull)
```

Reply:
good catch, my bad! fixed